packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -484,6 +485,7 @@ const Autocomplete = <
       onChange={onChange}
       onInputChange={onInputChange}
       onFocus={onFocus}
+      // openText={t("open.text")} // OKTA-747967 Uncomment this line after translation is received
```

Comment:
Why do we have to wait for translations to be received? Is this a requirement for odyssey?

Reply:
Good point, we can actually merge the code, no need to comment it out

At the end, today we have English. If we merge this code, we will still have English, so no change there. Once we deliver translations, then all is fixed and we won't need to uncomment this at that time.